src/SourceBuild/content/global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "9.0.100-rc.1.24452.12"
+    "dotnet": "10.0.100-alpha.1.24507.21"
```

Comment:
Undo this change. We can't build with a 10.0.100 SDK yet in the VMR.

Reply:
We can try applying https://github.com/dotnet/runtime/pull/109265.patch to see if there anything else blocking the flow.

Reply:
Not sure we can have the SDK target 10 and not use the net10 sdk? Do we need to disable the VMR until the support for the 10 sdk is enabled?

Reply:
True, if a repo that is part of the VMR builds with a newer sdk and targets a newer TFM, then the VMR's SDK needs to be updated as well.

I will look into this tomorrow. We might just need to set a few additional temporary global properties when building runtime.